tools/dynamatic/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ add_llvm_tool(dynamatic
   dynamatic.cpp
 )
 
+include_directories(/usr/include/readline)
```

Comment:
This is a standard location for headers so this shouldn't be necessary (it works on my machine without it).

Reply:
Done